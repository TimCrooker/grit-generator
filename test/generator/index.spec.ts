import path from 'path'
import { Projen } from 'projenerator'

const generator = path.join(__dirname, '..')

test('defaults', async () => {
	const projen = new Projen({
		generator,
		mock: true
	})
  await projen.run()

  expect(await projen.getOutputFiles()).toMatchSnapshot()
})
