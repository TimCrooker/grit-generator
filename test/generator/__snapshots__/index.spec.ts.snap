// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaults 1`] = `
Array [
  "generator/__snapshots__/index.spec.ts.snap",
  "generator/index.spec.ts",
]
`;
